addons/website_slides/models/slide_channel.py

```diff
@@ -345,23 +346,32 @@ def create(self, vals):
             vals['channel_partner_ids'] = [(0, 0, {
                 'partner_id': self.env.user.partner_id.id
             })]
+        if not vals.get('description_card') and vals.get('description'):
```

Comment:
I would check description first, as it's the first trigger to eventually modify description_card : 
`if vals.get('description') and not vals.get('description_card'): ...`